src/Tools/dotnet-monitor/dotnet-monitor.csproj

```diff
@@ -23,6 +23,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" Version="$(AzureStorageBlobsVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration.KeyPerFile" Version="$(MicrosoftExtensionsConfigurationKeyPerFileVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
```

Comment:
Is it a bad idea to pull in 5.0 packages into a 3.1 app?

Reply:
I don't think it's a problem. The packages are netstandard2.0 target.

Reply:
The 5.0.0 package is ns2.0 and netcoreapp3.0.

Reply:
Looks like Microsoft.Extensions.Logging.Console 5.0.0 pulls in Microsoft.Extensions.Logging 5.0.0 however the Microsoft.Diagnostics.Monitoring.EventPipe project references the 2.1.1 version and the Microsoft.Diagnostics.Monitoring.RestServer project references the 3.1.0 version (via the target framework). My brief testing shows that it hasn't been a problem and the debugger does show that the 5.0.0 version is ultimately loaded. It's likely that the logging and extensions libraries are backward compatible with their prior versions.

Reply:
Looking at the [API diff](https://www.fuget.org/packages/Microsoft.Extensions.Logging/5.0.0/lib/netstandard2.1/diff/3.1.11/) , it's been purely additive API surface between M.E.L 3.1.1 to 5.0.0. I'd just sanity check all the force upgraded assemblies to ensure that we aren't using any API that is no longer present.

Reply:
@maryamariyan If she's aware of any API breaks

Reply:
@shirhatti there is also an [API diff](https://www.fuget.org/packages/Microsoft.Extensions.Logging.Console/5.0.0/lib/netstandard2.1/diff/3.1.11/) for M.E.L.Console and here is a [link](https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/5.0/obsolete-consoleloggeroptions-properties) to the obsoleted properties